cypress/e2e/smoke/create_tx.cy.js

```diff
@@ -35,9 +35,15 @@ describe('Queue a transaction on 1/N', () => {
   })
 
   it('should create a queued transaction', () => {
+    // Spy the /estimations request
+    cy.intercept('POST', '/**/multisig-transactions/estimations').as('estimations')
+
     // Alias for New transaction modal
     cy.contains('h2', 'Review transaction').parents('div').as('modal')
 
+    // Wait for /estimations response
+    cy.wait('@estimations')
```

Comment:
Instead of waiting for the real request, I've also considered stubbing the first request by using `Date.now()` for the recommended nonce. This way we wouldn't have to hit the server with any `/estimations` request at all and we'd guarantee the nonce is never repeated.

It would make the test more robust but it felt like an overkill. WDYT?

Reply:
IMHO e2e tests should not stub anything.

Reply:
I've changed it. Only spy on the request now